### ISSUE TITLE
error: no suitable constructor found for FilterableNotification

### ISSUE THREAD

**Issue by Sewb21** [Plotprojects/plotprojects-react-native-module#6]
I am getting the following error when trying to build the latest package version (1.0.9). I have tested this using our own private repo and also your provided example project.

```
/Users/sebsmith/Code/plot-react-native-module-example/node_modules/plotprojects-react-native-module/android/src/main/java/com/plotprojects/retail/android/react/util/FilterableNotificationMarshaller.java:101: error: no suitable constructor found for FilterableNotification(String,Option<String>,String,int,String,String,String,double,double,Option<Integer>,String,String,int,int,int,boolean,Map<String,String>,int,String,Map<String,String>,Map<String,String>)
        return new FilterableNotification(campaignId,
               ^
    constructor FilterableNotification.FilterableNotification(String,String,String,String,String,double,double,String,int,int,String) is not applicable
      (actual and formal argument lists differ in length)
    constructor FilterableNotification.FilterableNotification(String,String,String,String,String,double,double,int,String,int,int,String) is not applicable
      (actual and formal argument lists differ in length)
    constructor FilterableNotification.FilterableNotification(String,Option<String>,String,int,String,String,String,double,double,Option<Integer>,String,String,int,int,int,boolean,Map<String,String>,int,String,Map<String,String>,Map<String,String>,String) is not applicable
      (actual and formal argument lists differ in length)
      ```

**Comment by m2jobe** [comment]
I am getting this too

**Comment by marshall20** [comment]
Thanks for the input. We're investigating the issue and will provide some feedback soon

**Comment by ddarren** [comment]
@marshall20 The issue is with newer version of the `com.plotprojects:plot-android:3.15` library. If I modify `ode_modules/plotprojects-react-native-module/android/build.gradle` and change `api('com.plotprojects:plot-android:3.15.+')` to `api('com.plotprojects:plot-android:3.15.1')` then it builds fine.

BTW, this is the second time I've had a similar issue with the react plot projects module, where having the android version be somewhat dynamic caused our builds to fail. I definitely recommend introducing some CI for this or even using a fully absolute version.

**Comment by marshall20** [comment]
@ddarren Indeed that solution works. We introduced a breaking change by mistake on a custom release to a specific client. We'll release a new version soon. Meanwhile @m2jobe  @Sewb21  please use this solution.

**Comment by yahya-plotprojects** [comment]
This issue should be fixed in version `1.0.11`
